README.md

```diff
@@ -112,115 +112,74 @@ On **Ubuntu**, you need to install the following packages:
-ninja
-ninja install
+./oqs-scripts/clone_liboqs.sh
+./oqs-scripts/build_liboqs.sh
 ```
 
-Building liboqs requires your system to have OpenSSL 1.1.1 or higher already installed. It will automatically be detected if it in a standard location, such as `/usr` or `/usr/local/opt/openssl@1.1` (for brew on macOS).  Otherwise, you may need to specify it with `-DOPENSSL_ROOT_DIR=<path-to-system-openssl-dir>` added to the `cmake` command.
+Building liboqs requires your system to have OpenSSL 1.1.1 or higher already installed. It will automatically be detected if it is under `/usr` or another standard location.  Otherwise, you may need to specify it with `-DOPENSSL_ROOT_DIR=<path-to-system-openssl-dir>` added to the `cmake` command.
 
 ### Step 2: Build the fork
 
-In `<OPENSSH_ROOT>`, first run:
+Run the following:
 
 ```
-export OPENSSH_INSTALL=<path-to-install-openssh>
-autoreconf
+env OPENSSL_SYS_DIR=<PATH_TO_OPENSSL> ./oqs-scripts/build_openssh.sh
 ```
 
-Then, run the following:
-
-	./configure --with-ssl-dir=<path-to-openssl>/include \
-	            --with-ldflags=-L<path-to-openssl>/lib   \
-	            --with-libs=-lm                          \
-	            --prefix=$OPENSSH_INSTALL                \
-	            --sysconfdir=$OPENSSH_INSTALL            \
-	            --with-liboqs-dir=`pwd`/oqs
-	make 
-	make install
+`OPENSSL_SYS_DIR` does not have to be specified if OpenSSL is present under `usr/`.
 
-Again, the `path-to-openssl` (1.1.1) does not need to be specified if it is in one of the standard locations.
+As not all tests in the stock regression suite pass, run `oqs-test/run_tests.sh` instead of simply executing `make tests` to ensure the build was successful.
 
-So, in summary, if OpenSSL is installed in a default location and `oqs-openssh` is to be installed in `/opt/openssh` this command builds and installs `oqs-openssh`: `export OPENSSH_INSTALL=/opt/openssh && autoreconf && ./configure --with-libs=-lm --prefix=$OPENSSH_INSTALL --sysconfdir=$OPENSSH_INSTALL --with-liboqs-dir=`pwd`/oqs && make && make install`
-
-As not all stock `openssh` tests are passing, be sure to execute `oqs-test/run_tests.sh` instead of simply executing `make tests` to ensure the build was successful.
-
-To execute a connection test with one of the supported quantum-safe algorithms (chosen at random), run `python3 oqs-test/try_connection.py`. If all algorithms should be exercized, pass a parameter to this command, e.g., like this: `python3 oqs-test/try_connection.py all`. Be aware that this test can take a long time due to the number of algorithm combinations available.
+To execute a connection test with a randomly chosen key-exchange and signature algorithm, run `python3 oqs-test/try_connection.py`. If it is desired that all such combinations should be tested, run `python3 oqs-test/try_connection.py all`. Be aware that this test can take a long time due to the number of algorithm combinations available.
```

Comment:
Maybe worthwhile adding a sentence that the "doall" parameter executes each SIG and KEM _exactly once_?

Reply:
Added a parenthetical comment.